### PR TITLE
sbd integration into tx5

### PR DESCRIPTION
**Pull request by neonphog**
DEPENDS ON: https://github.com/holochain/sbd/pull/1
RESOLVES: https://github.com/holochain/sbd/issues/17

- deletes tx5-signal-srv - we will now be using sbd-server
- updates tx5-signal to be a thin wrapper around sbd-e2e-crypto-client
- adds crate tx5-connection which manages connections initially talking over the signal connection (and a placeholder to eventually upgrade to webrtc connections to be added in a follow-on PR)
- updates tx5 endpoint to work with tx5-connection-s

### CODE REVIEW

File: crates/tx5-connection/src/conn.rs [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                            // invalid
+                            HandshakeReq(_) | HandshakeRes(_) => break,
+                            Message(msg) => {
+                                if msg_send.send(msg).await.is_err() {
+                                    break;
+                                }
+                            }
+                            _ => (),
+                        }
+                    }
```
Comment (neonphog) [inline crates/tx5-connection/src/conn.rs:155]: Here's the webrtc placeholder. Currently we're just dealing with the `Message` type, but eventually we'll pay attention to offers/answers/ice and try to establish a webrtc connection, switching to it if it is successful.

File: crates/tx5/src/ep.rs [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    pub fn get_stats(&self) -> stats::Stats {
+        #[cfg(feature = "backend-go-pion")]
+        let backend = stats::StatsBackend::BackendGoPion;
+        #[cfg(feature = "backend-webrtc-rs")]
+        let backend = stats::StatsBackend::BackendWebrtcRs;
+
+        stats::Stats {
+            backend,
+            peer_url_list: self.get_listening_addresses(),
+            connection_list: Vec::new(),
```
Comment (neonphog) [inline crates/tx5/src/ep.rs:285]: Note, this just returns an empty array for now. I'll flesh out the stats once we have the webrtc connections working.

File: .github/workflows/stale.yml
Comment (ThetaSinner) [inline .github/workflows/stale.yml:1]: Just not useful on this repo?

File: Cargo.toml [hunk 3]
```diff
@@ -15,7 +15,7 @@ resolver = "2"
 [workspace.dependencies]
 app_dirs2 = "2.5.5"
 backtrace = "0.3.69"
-base64 = "0.13.0"
+base64 = "0.22.1"
```
Comment (ThetaSinner) [inline Cargo.toml:18]: 🥳

File: .github/workflows/stale.yml
Comment (neonphog) [reply .github/workflows/stale.yml:1]: exactly. The volume isn't high enough to warrant it, and it doesn't have the permanent label... if we ever need it, we can port the one from holochain over here.

**Review by ThetaSinner — APPROVED** [review]
This is large as advertised. I think I'm following most of the changes and they make sense. I'm calling this a 'high level' approval where I think the structure of the changes and the code looks good. If there are logic errors in loops or task management etc then I'm probably not going to catch it in this much code :)

**Pull request merged**